### PR TITLE
upgrade bootstrap and react-bootstrap to latest, update site components

### PR DESCRIPTION
**Pull request by jimmcgaw**

### CODE REVIEW

**Comment by elimisteve** [comment]
Nice! There have been a couple remaining old dependencies and this should help a bunch with that.

**Comment by elimisteve** [comment]
These larger fonts look fantastic btw

File: src/components/general/AlertContainer.js [hunk 1]
```diff
@@ -29,8 +29,9 @@ const AlertContainer = ({
   return (
     <div className="alert-container" style={{marginRight: '10px'}}>
       {alertMessage && <Alert
-        bsStyle={alertStyle}
-        onDismiss={dismissAlert}>
+        variant={alertStyle}
+        onClose={() => dismissAlert()}
```
Comment (elimisteve) [inline src/components/general/AlertContainer.js]: This is small, but unless we're aiming to get more extensibility out of doing

```js
onClose={() => {
  // Can add more shit here
  dismissAlert();
  // ...or here
}
```

then we'd might as well turn lines like

```js
onClose={() => dismissAlert()}
```

into

```js
onClose={dismissAlert}
```

Yeah? Slightly more efficient by creating one less function?

Comment (jimmcgaw) [reply src/components/general/AlertContainer.js]: Yep, I can update this. I forget why I ended up at an anonymous arrow function here anyway.

File: src/store/actions/alertActions.js [hunk 2]
```diff
@@ -4,10 +4,10 @@ export const ALERT_DISMISS = 'ALERT_DISMISS';
 export const alertSuccess = (message, alertRenderSeconds) =>
   ({ type: ALERT_DISPLAY, style: 'success', message, alertRenderSeconds });
 
-export const alertWarning = (message) =>
+export const alertWarning = (message, alertRenderSeconds) =>
   ({ type: ALERT_DISPLAY, style: 'warning', message, alertRenderSeconds });
 
-export const alertDanger = (message) =>
+export const alertDanger = (message, alertRenderSeconds) =>
   ({ type: ALERT_DISPLAY, style: 'danger', message, alertRenderSeconds });
```
Comment (jimmcgaw) [inline src/store/actions/alertActions.js:11]: Also fixed these.

File: README.md [hunk 3]
```diff
@@ -241,9 +289,9 @@ sudo setcap cap_net_bind_service=+ep leapchat
 
 ## Documentation Links
 
-Currently, some of the packages we use are slightly out of date. Here are some quick documentation links that might prove useful:
+[Bootstrap 5](https://getbootstrap.com/docs/5.0/getting-started/introduction/)
 
-[React Bootstrap version 0.33.1 docs](https://react-bootstrap-v3.netlify.app/components/tooltips/)
+[React Bootstrap 2](https://react-bootstrap.github.io/)
```
Comment (jimmcgaw) [inline README.md]: These links are the only update we really need to go along with this branch. The rest I've just been meaning to do for a while.

File: src/components/general/AlertContainer.js (hunk 1 above)
Comment (elimisteve) [reply src/components/general/AlertContainer.js]: :+1:

File: README.md [hunk 4]
```diff
@@ -241,11 +289,13 @@ sudo setcap cap_net_bind_service=+ep leapchat
...
-Currently, some of the packages we use are slightly out of date. Here are some quick documentation links that might prove useful:
-
-[React Bootstrap version 0.33.1 docs](https://react-bootstrap-v3.netlify.app/components/tooltips/)
+Open via `npm`:
 
-Search for available [Font Awesome Icons](https://fontawesome.com/v5/search)
+```
+$ npm docs bootstrap
+$ npm docs react-bootstrap
+$ npm docs react-icons
```
Comment (jimmcgaw) [inline README.md]: This makes a good deal more sense than keeping the links themselves up-to-date.

File: src/store/actions/alertActions.js (hunk 2 above)
Comment (elimisteve) [reply src/store/actions/alertActions.js:11]: :+1:

**Comment by elimisteve** [comment]
@jimmcgaw Looks like use of `bsStyle` was accidentally re-added to this branch in the last force-push?

EDIT: No I just checked out the wrong branch locally without running `npm i`; nevermind.

File: README.md (hunk 4 above)
Comment (elimisteve) [reply README.md]: Ha, sounds good

File: README.md [hunk 5]
```diff
@@ -52,14 +52,53 @@ Then grab and build the `leapchat` source:
...
+from the browser without any CORS configuration.
 
 ### macOS Instructions
 
 If you don't already have Postgres 9.5 to Postgres 12 installed and
 running, install it with Homebrew:
 
 ```
-brew install postgresql@10
+brew install postgresql@12
```
Comment (elimisteve) [inline README.md:101]: `$ ` prefix?

File: README.md [hunk 6]
```diff
@@ -70,23 +109,37 @@ Next, you'll need three terminals.
 **In the first terminal**, run database migrations, download `postgrest`,
 and have `postgrest` connect to Postgres:
 
+Start the PostgreSQL server:
+
+```
+brew services start postgresql@12
```
Comment (elimisteve) [inline README.md:115]: `$ ` ftw?

File: README.md (hunk 5 above)
Comment (elimisteve) [reply README.md:101]: Or we remove the `$`s so it's easy to copy and paste the lines into their shall; that's what I've moved to even though that indeed makes it less clear that these are commands.

**Review by elimisteve — APPROVED** [review]
LGTM :ok_hand:

Mentioned a trivial thing, but let's merge and we can fuck with the `README` command formatting later; no need to block.

**Comment by elimisteve** [comment]
@jimmcgaw Ready to merge? LGTM

**Comment by elimisteve** [comment]
_Dooo iiit_

**Comment by elimisteve** [comment]
:racehorse: :racehorse: :racehorse: :racehorse: :racehorse:

**Pull request merged**